src/services/Asn.ts

```diff
@@ -37,6 +41,51 @@ class Asn {
       this.sequence ?? "".toString().padStart(11, "0")
     }${this.checkCharacter()}`
   }
+
+  static yearChars = 2
+
+  static forceAndUnitChars = 4
+
+  static systemChars = 2
```

Comment:
Suggestion: I would put the magic numbers after the `class Asn` and before `public asn: string`. I think it's more typical for languages.

Reply:
Rearranged variables